sdk/spring/azure-spring-boot-starter-active-directory/README.md

```diff
@@ -230,8 +208,37 @@ public class AADOAuth2ResourceServerSecurityConfig extends WebSecurityConfigurer
 }
 ```
 
-### [Web API] OAuth 2.0 On-Behalf-Of flow
-Please refer [azure-spring-boot-sample-active-directory-resource-server-obo] to for access On-Behalf-Of flow.
+### Resource server access other resources
+
+#### System diagram
+Resource server accesses other resource servers which are protected by Azure AD.
+![Resource Server Access Other Resources](resource/add-based-resource-server-access-other-resources.png)
```

Comment:
There is a mistake in this picture.

![image](https://user-images.githubusercontent.com/13167207/106098129-7d860300-6173-11eb-8a34-319cafc4b362.png)

It should be `Resource Server` instead of `Web application`.

Reply:
![image](https://user-images.githubusercontent.com/13167207/106098272-b45c1900-6173-11eb-824c-8ca4d7d607d8.png)
Just use one resource server, named `Other resource server` will make it easier to understand.